### ISSUE TITLE
show also OSM notes

### ISSUE THREAD

**Issue by matkoniecz** [keepright/keepright#54]
OSM notes frequently point out problems not detected by automatic data processing

**Comment by mueschel** [comment]
You are right, this would be a nice enhancement. Unfortunately I don't know how this could be done. If anybody describes how this could be done, I'm happy to include it.
